content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,310 @@
 
-When opening the detailed operations, you can click on *Add a line*.
-Then, you will be able to fill in the serial number field. Once done,
-you can click again on *Add a line* to register a new serial number.
+On this :abbr:`RFQ (request for quotation)` form, fill out the necessary information, by adding a
+:guilabel:`Vendor`, and by adding the desired products to the :guilabel:`Product` lines, by clicking
+:guilabel:`Add a product`, under the :guilabel:`Products` tab.
 
-.. image:: serial_numbers/serial_numbers_05.png
-    :align: center
+Choose the desired quantity of the product to order, by changing the number in the
+:guilabel:`Quantity` column.
 
-Multi-assignation
-~~~~~~~~~~~~~~~~~
+When the necessary configurations are complete, click :guilabel:`Confirm Order`. This will convert
+the :abbr:`RFQ (request for quotation)` to a purchase order.
 
-By using the multi-assignation of serial numbers, Odoo will
-automatically create the necessary lines. To do so, you have to enter
-the first serial number of your set and the number of products you have
-to assign a serial number to.
+Then, click the :guilabel:`Receipt` smart button to be taken to the warehouse receipt form page for
+that specific :abbr:`PO (purchase order)`.
 
-Once done, click on *Assign Serial Numbers* and Odoo will do the rest.
+.. note::
+   Clicking :guilabel:`Validate` before assigning a serial number to the ordered product quantities
+   will cause a :guilabel:`User Error` pop-up to appear. The pop-up requires entry of a lot or
+   serial number for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be
+   validated without a serial number being assigned.
 
-.. image:: serial_numbers/serial_numbers_06.png
-    :align: center
+.. image:: serial_numbers/serial-numbers-user-error-popup.png
+   :align: center
+   :alt: User error popup prompting serial number entry.
 
-.. image:: serial_numbers/serial_numbers_07.png
-    :align: center
+From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`hamburger
+(four horizontal lines)` icon, located to the right of the :guilabel:`Unit of Measure` column in the
+:guilabel:`Operations` tab. Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
 
-Copy/pasting from an Excel file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In this pop-up, configure a number of different fields, including the assignation of a serial number
+(or serial numbers) under the :guilabel:`Lot/Serial Number Name` column, located at the bottom of
+the pop-up.
 
-To use copy/pasting, open the spreadsheet containing the serial numbers
-you received and copy the list. Then, past them in the *Lot/Serial
-Number Name* column of the wizard. Doing so, Odoo will automatically
-create the lines you need.
+There are three ways to do this: manually assigning serial numbers, automatically assigning serial
+numbers, and copy/pasting serial numbers from an Excel file.
```

Comment:
does it *have* to be an *Excel* file?
If so we should italicize this since it's an MS product name.
If not, we can simply say "spreadsheet"